Lib/ufo2ft/fontInfoData.py

```diff
@@ -311,8 +311,8 @@ def postscriptWeightNameFallback(info):
     openTypeVheaCaretOffset=None,
 
     postscriptUniqueID=None,
-    postscriptUnderlineThickness=None,
-    postscriptUnderlinePosition=None,
+    postscriptUnderlineThickness=50,
+    postscriptUnderlinePosition=-75,
```

Comment:
> AFDKO assumes that upem=1000

Maybe for these underline values, but not for the others addressed in this PR. But that's probably a bug in the AFDKO. I'll fix this here.